packages/council-graphql/package.json

```diff
@@ -29,7 +29,7 @@
     "prettier:check": "prettier --check src/**/*.{gql,graphql,js,jsx,ts,tsx,json,md}"
   },
   "lint-staged": {
-    "**/*.{gql,graphql,js,jsx,ts,tsx,json,md}": [
+    "src/**/*.{gql,graphql,js,jsx,ts,tsx,json,md}": [
```

Comment:
Can we add `dist/` to .eslintignore and .prettierignore instead? I'd like for our top-level config files to be prettiered as well, ie: `tsconfig.json` etc..